File: GlasspaperSolutions.Client/components/Resource.js
import useGetResourceById from '@/hooks/useGetResourceById'

export default function Resource({resource}) {
    const [ resourceById ]  = useGetResourceById(resource?.resourceId)

    return (
        <>
        <div>
            <p>{resourceById?.name}</p>
        </div>
        </>    
    )
}

File: GlasspaperSolutions.Model/Resource.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GlasspaperSolutions.Model
{
    public class Resource {
        [Key]
        public int ResourceId { get; set; }
        [Required]
        public string? Name { get; set; }
        [Required]
        public string? Category { get; set; }
        [Required]
        public ICollection<BookingResource>? Bookings { get; set; }
    }
}

